app/models/manageiq/providers/azure/cloud_manager.rb

```diff
@@ -1,3 +1,5 @@
+require 'azure-armrest'
```

Comment:
I was hitting some kind of weird load ordering issue that I couldn't nail down, which resulted in undefined constant errors.